test/gtest/uct/ib/test_ib.cc

```diff
@@ -212,7 +212,7 @@ class test_uct_ib : public uct_test {
 
         uct_ib_address_unpack(ib_addr, &lid_out, &gid_out);
 
-        if (IBV_PORT_IS_LINK_LAYER_ETHERNET(uct_ib_iface_port_attr(iface))) {
+        if (uct_ib_iface_is_roce(iface)) {
```

Comment:
minor: there is another `IBV_PORT_IS_LINK_LAYER_ETHERNET` on line 167

Reply:
i can't change the one in line 167 because it gets port_attr and not ib_iface